content/docs/guides/crosswalk/aws/vpc.md

```diff
@@ -38,11 +38,11 @@ used for strong network isolation between endpoints and resources that are other
 
 ## Managing VPCs
 
-The [aws.ec2.Vpc]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Vpc" >}}) class provides full access to the
-AWS VPC API, and [aws.ec2]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2" >}}) the entire AWS EC2 API. Using
+The [aws.ec2.Vpc]({{< relref "/docs/reference/pkg/aws/ec2/vpc/" >}}) class provides full access to the
+AWS VPC API, and [aws.ec2]({{< relref "/docs/reference/pkg/aws/ec2" >}}) the entire AWS EC2 API. Using
 these packages, you can configure all aspects of AWS networks for your applications and infrastructure.
 
-The [awsx.ec2.Vpc]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx/ec2#Vpc" >}}) class encapsulates a complete
+The [awsx.ec2.Vpc]({{< relref "/docs/reference/pkg/aws/ec2/vpc/" >}}) class encapsulates a complete
```

Comment:
This particular sentence is referring to `awsx`, so let's put this back to its previous value.

Reply:
oh, thank you, good catch